latex/non-integral.tex

```diff
@@ -2,7 +2,7 @@ \section{Non-Integral Calculations}
 \label{sec:non-integr-calc}
 
 In the ledger there are several cases where non-integral calculations are
-required. This does concern the delegation transitions, not value transactions.
+required, particularly calculations relating to delegation transitions.
```

Comment:
@JaredCorduan what do you think of this?

Reply:
perfect!